vm/ubpf_loader.c

```diff
@@ -220,13 +218,63 @@ ubpf_load_elf(struct ubpf_vm* vm, const void* elf, size_t elf_size, char** errms
                 goto error;
             }
 
-            unsigned int imm = ubpf_lookup_registered_function(vm, sym_name);
-            if (imm == -1) {
-                *errmsg = ubpf_error("function '%s' not found", sym_name);
+            switch (ELF64_R_TYPE(r.r_info)) {
+            // Perform map relocation.
+            case R_BPF_64_64: {
```

Comment:
Could we define our own internal type constants -- that way we can make one for the `2` below (that is sticking out like a sore thumb to me).

Reply:
Thank you!